maat-court-data-api/src/main/java/gov/uk/courtdata/repository/ContributionFilesRepository.java

```diff
@@ -0,0 +1,21 @@
+package gov.uk.courtdata.repository;
+
+import gov.uk.courtdata.entity.ContributionFilesEntity;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.stereotype.Repository;
+
+import java.time.LocalDate;
+import java.util.List;
+
+@Repository
+public interface ContributionFilesRepository extends JpaRepository<ContributionFilesEntity, Integer> {
+
+    @Query(value = "SELECT cf.xml_content FROM TOGDATA.CONTRIBUTION_FILES CF WHERE CF.FILE_NAME like '%%CONTRIBUTIONS%%' AND CF.DATE_CREATED BETWEEN fromDate and toDate",
```

Comment:
Are you sure we should be using the `xml_content` field here? Based on the description in the Controller "Get a list of XML files" I would have expected the field being used would be `file_name`?

i.e. for record with id 222687962 we have:
file_name = CONTRIBUTIONS_201810121932.xml
and
xml_content = ```
<?xml version="1.0"?>
<contribution_file>
  <header id="222687962">
    <filename>CONTRIBUTIONS_201810121932.xml</filename>
    <dateGenerated>2018-10-12</dateGenerated>
    <recordCount>12</recordCount>
    <formatVersion>format version 1.7 - xsd=contribution_file.xsd version 1.16</formatVersion>
  </header>
  <CONTRIBUTIONS_LIST>
    <CONTRIBUTIONS id="222684987" flag="new">
      <maat_id>5633879</maat_id>
      <applicant id="222684939">
        <firstName>george</firstName>
        <lastName>cantwell</lastName>
        <dob>1981-07-10</dob>
        <noFixedAbode>no</noFixedAbode>
        <specialInvestigation>no</specialInvestigation>
        <homeAddress>
          <detail>
            <line1>London</line1>
            <line2/>
            <line3/>
            <city/>
........

```